samples/bluetooth/handsfree_ag/boards/mimxrt1170_evk_mimxrt1176_cm7_B.conf

```diff
@@ -0,0 +1,6 @@
+#select NXP BT controller
+CONFIG_BT_NXP_CTLR=y
+#select NXP NW612 Chipset
+CONFIG_BT_NXP_NW612=y
+#select CRC, it is depended by CONFIG_BT_NXP_CTLR
+CONFIG_CRC=y
```

Comment:
Why not select it there then?

Reply:
Current, `BT_NXP_CTLR` use 'depend on' to require `CRC`. 

It is changed from "depend on" to "select".